src/scripts/config/config_test.go

```diff
@@ -198,3 +199,60 @@ func TestLoadEnvFromFile(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertFileToCRLF(t *testing.T) {
+	tests := []struct {
+		description     string
+		fileContent     string
+		expectedErr     bool
+		expectedContent string
+	}{
+		{
+			description:     "FileWithLF",
+			fileContent:     "line1\nline2\nline3",
+			expectedErr:     false,
+			expectedContent: "line1\r\nline2\r\nline3",
+		},
+		{
+			description:     "FileWithCRLF",
+			fileContent:     "line1\r\nline2\r\nline3",
+			expectedErr:     false,
+			expectedContent: "line1\r\nline2\r\nline3",
+		},
+		{
+			description:     "EmptyFile",
+			fileContent:     "",
+			expectedErr:     false,
+			expectedContent: "",
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			filePath := "test_file.txt"
+
+			// Creating and writing to a test file
+			err := os.WriteFile(filePath, []byte(test.fileContent), 0644)
+			if err != nil {
+				t.Fatalf("Couldn't create test file: %v", err)
+			}
+			defer os.Remove(filePath)
```

Comment:
this is likely best encapsulated in a `t.Cleanup`

```suggestion
			t.Cleanup(func() {
				_ = os.Remove(filePath)
			}
```